### PR TITLE
ADEN-12438 Create a bundle for GameFAQs and Gamespot

### PR DESCRIPTION
**Pull request by szymon-nawrocki**
### Ticket
https://fandom.atlassian.net/browse/ADEN-12438

### CODE REVIEW

File: src/platforms/gamefaqs/ad-context.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const basicContext = {
+	adUnitId: '/{custom.dfpId}/aw-gamefaqs',
```
Comment (nandy-andy) [inline src/platforms/gamefaqs/ad-context.ts:2]: Something to be addressed in the future probably. We could move `'aw'` to `custom` under `region` field so it follows the current config (`dfpPath.region`). But I've never seen anything else than `'aw'`, so probably it's not used and therefore we could address it in the future ;)

File: src/platforms/gamefaqs/setup/context/slots/gamefaqs-slots-context.setup.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+			leader_plus_top: {
+				defaultSizes: [
+					[728, 90],
+					[970, 66],
+					[970, 250],
+				],
+			},
+			mpu_top: {
+				defaultSizes: [[300, 250]],
+			},
```
Comment (nandy-andy) [inline src/platforms/gamefaqs/setup/context/slots/gamefaqs-slots-context.setup.ts:17]: I was thinking if we could add here all the slot names and sizes from the current config (look for `"units"` node in the JSON).

File: src/platforms/gamefaqs/setup/dynamic-slots/gamefaqs-dynamic-slots.setup.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { context, DiProcess } from '@wikia/ad-engine';
+
+export class GamefaqsDynamicSlotsSetup implements DiProcess {
+	execute(): void {
+		this.wrapActiveAdSlots();
+	}
+
+	private wrapActiveAdSlots(): void {
```
Comment (nandy-andy) [inline src/platforms/gamefaqs/setup/dynamic-slots/gamefaqs-dynamic-slots.setup.ts:8]: It's not really wrapping but injecting, so I'd rename the method :) In more advanced code of ours (ucp_mobile or ucp_desktop) you can actually find `injectSlots()` function being used in the dynamic-slots but I wonder if we could go with something simpler first 😊 If we get to an obstacle easily solved by adding similar logic to UCP bundles we'd go there but maybe before we could rethink the whole slots/dynamic slots setup again ;)

File: src/platforms/gamefaqs/gamefaqs-platform.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+export class GamefaqsPlatform {
+	constructor(private pipeline: ProcessPipeline) {}
+
+	execute(): void {
+		this.pipeline.add(
+			// TODO: we need a CMP step here, so we won't call for ads unless we have a clear idea of the privacy policy of a visitor
+			// TODO: to decide if we want to call instant-config service for the first releases?
+			GamefaqsSlotsContextSetup,
+			GamefaqsDynamicSlotsSetup,
+			// TODO: add targeting setup once we have idea of page-level and slot-level targeting
```
[the remark this answers is not in the document]
Comment (szymon-nawrocki) [reply src/platforms/gamefaqs/gamefaqs-platform.ts:20]: I think I have done it by adding `targeting.pos` to file: src/platforms/gamefaqs/setup/context/slots/gamefaqs-slots-context.setup.ts

File: src/platforms/gamefaqs/setup/context/slots/gamefaqs-slots-context.setup.ts (hunk 2 above)
Comment (szymon-nawrocki) [reply src/platforms/gamefaqs/setup/context/slots/gamefaqs-slots-context.setup.ts:508]: done ✅

File: src/platforms/gamefaqs/gamefaqs-platform.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (nandy-andy) [reply src/platforms/gamefaqs/gamefaqs-platform.ts:20]: Replied offline ;)

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**